pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -358,6 +358,12 @@ type WorkflowSpec struct {
 
 	// ArchiveLogs indicates if the container logs should be archived
 	ArchiveLogs *bool `json:"archiveLogs,omitempty" protobuf:"varint,40,opt,name=archiveLogs"`
+
+	// PauseBefore indicates which of the nodes that should be pause before execution to allow debugging
+	PauseBefore map[string]string `json:"pauseBefore,omitempty" protobuf:"bytes,41,opt,name=pauseBefore"`
```

Comment:
do we need these, I think you could specify the env vars in the template spec?

Reply:
Good point, will update.